namespaces/collectd.yml

```diff
@@ -146,11 +156,6 @@ namespace:
       Corresponds to collectd's disk plugin.
       Or virt plugin with types:disk_octets/disk_ops.
     fields:
-    - name: disk_name
```

Comment:
This require further manipulation for each plugin separately.
I think it will be best to leave as is in the "Plugin_instance" field.